### ISSUE TITLE
Alot of the games if not all of them not working.

### ISSUE THREAD

**Issue by Lhinger** [liamhtml/PLAYHYDRA#4]
On the website, almost all of the games either dont display a game at all or show the chrome error and say "connection reset"

Could it be that the player its using is what is causing this?

**Comment by Lhinger** [comment]
I looked into it a bit and apparently, the reason one of the games (Drift hunters) is blocked is that the game is not local to the site and is just playing it off of a website called debbiewatermanphd.com

**Comment by liamhtml** [comment]
Yes, I'm aware of this and I'm working to fix it. Hadn't heard of drift hunters though so thanks for the tip.

**Comment by liamhtml** [comment]
debbiewatermanphd.com is what I believe to be a normal website that its hosting provider also sells file hosting space on for random websites. there are a good few games hosted there.
